browser_patches/firefox-beta/BUILD_NUMBER

```diff
@@ -1,2 +1,2 @@
-1300
-Changed: yurys@chromium.org Wed Oct 27 17:58:07 PDT 2021
+1301
+Changed: dg@domgan.com Mon 1 Nov 2021 13:04:12 GMT
```

Comment:
looks like the line ending has changed here (we usually use the script to update the file (browser_patches/export.sh).